modules/src/main/java/org/archive/modules/extractor/ExtractorHTML.java

```diff
@@ -43,1000 +45,999 @@
 import org.springframework.beans.factory.InitializingBean;
 import org.springframework.beans.factory.annotation.Autowired;
 
+import com.google.common.base.Strings;
+
+import au.id.jericho.lib.html.Element;
```

Comment:
I think this patch needs to be cleaned up a bit before it can be merged. It seems to be modifying nearly every line in the file? It's also introducing a bunch of unused imports? I haven't reviewed the actual code changes as it's too hard to see what changed. :(